src/open_inwoner/openzaak/import_export.py

```diff
@@ -27,16 +27,17 @@
 
 class ZGWImportError(Exception):
     @classmethod
-    def extract_error_data(cls, exc: Exception, jsonl: str):
-        exc_source = type(exc.__context__)
+    def extract_error_data(cls, exception: Exception | None = None, jsonl: str = ""):
         data = json.loads(jsonl) if jsonl else {}
```

Comment:
Let's use `None` in case an empty string is actually passed and we want to error on _that_ as well:

```suggestion
    def extract_error_data(cls, exception: Exception | None = None, jsonl: str | None= None):
        data = json.loads(jsonl) if jsonl is not None else {}
```